src/masonite/request.py

```diff
@@ -89,6 +91,29 @@ def input(self, name, default=False, clean=False, quote=True):
 
         return clean_request_input(self.request_variables.get(name, default), clean=clean, quote=quote)
 
+    def query_param(self, name, default=None, multi_value=False):
+        """Get a specific query string value.
+
+        Arguments:
+            name {string} -- Key of the input data
+
+        Keyword Arguments:
+            default {any} -- Default value if input does not exist (default: {None})
+            first {bool} -- Whether to return all values of a multi value query string param
+                            ie. quuery_param=one&query_param=two (default: {False})
+
+        Returns:
+            any
+        """
+        try:
+            value = self.query_params[name]
```

Comment:
This block can simply go to `value = self.query_params.get(name, default)`

Reply:
If not multi_value, we return value [0] which is not what we want to do for default [0]. default is to be returned verbatim.